shared/teams/team/tabs/container.tsx

```diff
@@ -24,6 +24,7 @@ export default Container.connect(
       _members,
       admin: yourOperations.manageMembers,
       error: state.teams.errorInAddToTeam,
+      isBig: Constants.isBigTeam(state, teamMeta.teamname),
```

Comment:
This isn't working for me, the "Channels" tab isn't showing up for team `zila.test`.

Reply:
You have to go to the team in chat first. We have a ticket to fix it I think, I felt this PR was too big already

Reply:
The only way I could get this working is by removing the `isBig` check. I tried all different paths of navigation and couldn't quite get there.

Reply:
Hmm... I found that going to a channel of the team in chat would load the requisite state, and then when going to the team page it would be there.

Reply:
Ah, you meant actually visiting a channel. Got it. Works now.